istioctl/pkg/precheck/precheck.go

```diff
@@ -122,10 +115,126 @@ See %s for more information about causes and resolutions.`, url.ConfigAnalysis)
 		fmt.Sprintf("The severity level of precheck at which to display messages. Valid values: %v", diag.GetAllLevelStrings()))
 	cmd.PersistentFlags().StringVarP(&msgOutputFormat, "output", "o", formatting.LogFormat,
 		fmt.Sprintf("Output format: one of %v", formatting.MsgOutputFormatKeys))
+	cmd.PersistentFlags().StringVarP(&fromCompatibilityVersion, "from-version", "f", "",
+		"check changes since the provided version")
 	opts.AttachControlPlaneFlags(cmd)
 	return cmd
 }
 
+func checkFromVersion(ctx cli.Context, revision, version string) (diag.Messages, error) {
```

Comment:
Technically it could impact anything, not just networking. Although networking happens to be responsible for the 4 flags in this PR..

Reply:
> Technically it could impact anything

Agree with this. Maybe just leave with the current settings, as the person who wrote this specific configuration should also be familiar with it.